lib/recurly/hosted-fields.js

```diff
@@ -160,8 +160,14 @@ export class HostedFields extends Emitter {
     if (destination) destination.focus();
   }
 
+  /**
+   * Provides an Array of tabbable Elements in the document body,
+   * excluding those which correspond to hosted field iframes
+   * @return {Array} Elements
+   */
   tabbableItems () {
-    return tabbable(window.document.body);
+    const fieldFrames = this.fields.map(f => f.iframe);
+    return tabbable(window.document.body).filter(el => !~fieldFrames.indexOf(el));
```

Comment:
I'm not sure how I feel about the use of `!~` for readability, it's somewhat obscure. But your comment at the top of the function explains what its doing so it's not too bad.

Reply:
It's a hold-over from IE11 not supporting `includes`, peppered throughout the codebase. I'm very excited to remove it in the future.